scipy/stats/tests/test_distributions.py

```diff
@@ -1377,6 +1377,34 @@ def test_sf_isf(self):
         xx = stats.gumbel_l.isf(y)
         assert_allclose(x, xx)
 
+        
+class TestGammaGompertz(object):
+    def test_pdf(self):
+        vals = stats.gamma_gompertz.pdf(np.log(np.arange(1., 6., 1.)), 1., 2.)
+        expected = [0.5, 4./9., 0.375, 8./25., 5./18.]
+        assert_almost_equal(vals, expected)
+        
+    def test_cdf(self):
+        vals = stats.gamma_gompertz.cdf(np.log(np.arange(1., 6., 1.)), 1., 2.)
+        expected = [0., 1./3., 0.5, 0.6, 2./3.]
+        assert_almost_equal(vals, expected)
+        
+    def test_ppf(self):
+        vals = stats.gamma_gompertz.ppf([0., 1./3., 0.5, 0.6, 2./3.], 1., 2.)
+        expected = np.log(np.arange(1., 6., 1.))
+        assert_almost_equal(vals, expected)
+        
+    def test_expon(self):
+        # test against expon (sepcial case for beta=1)
```

Comment:
typo: special -> special.

Reply:
Thanks for your suggestion. Accordingly,
```
        # test against expon (sepcial case for beta=1)
```
needs to be changed to
```
        # test against expon (special case for beta=1)
```